src/lib.rs

```diff
@@ -1,7 +1,4 @@
-#![no_std]
-
-#[cfg(any(test, feature = "std"))]
-extern crate std;
```

Comment:
This is intentional, see: https://www.reddit.com/r/rust/comments/1hs6spy/psa_for_std_feature_in_no_std_libraries/